src/applications/claims-status/containers/DetailsPage.jsx

```diff
@@ -45,10 +45,11 @@ class DetailsPage extends React.Component {
   setTitle() {
     const { claim } = this.props;
 
-    if (claim) {
```

Comment:
Fixed bug in DetailsPage.jsx . We were checking if the claim existed and then setting the document title. Changed it to instead check if there were claim attributes.